### ISSUE TITLE
flux:checkbox.all no longer checked since 1.0.20

### ISSUE THREAD

**Issue by seabasss** [livewire/flux#632]
Since version `1.0.20`, if all checkboxes are checked on initial load, the "select all" checkbox is no longer selected. I could add the checked attribute to all checkbox items to fix it, but it seems that something that worked in version `1.0.19` is no longer functioning as expected.

```php
// public array $checkboxes = ['one', 'two', 'three'];

<flux:checkbox.group wire:model.live="checkboxes">
    <flux:checkbox.all label="ALL" />

    <flux:checkbox value="one" label="One" />
    <flux:checkbox value="two" label="Two" />
    <flux:checkbox value="three" label="Three" />
</flux:checkbox.group>
```

1.0.20 and later:
![Image](https://github.com/user-attachments/assets/d5e89d46-93fe-4a01-8ca2-f0052d590127)

1.0.19
![Image](https://github.com/user-attachments/assets/a4c3ef38-eefd-49f4-981f-709b1990cea1)

**Comment by seabasss** [comment]
> [@seabasss](https://github.com/seabasss) Your code works as expected for me.
> 
> Have you updated to v1.0.22, `php artisan view:clear` and deleted any previously published version of the Flux checkbox components?

I just upgraded to `1.0.22` and I'm still experiencing the same issue. I haven't published any components, and I've cleared the views. I also tried different browsers, but the result is the same. Reverting to `1.0.19` restores the checkbox functionality. The "select all" checkbox shows `aria-checked="false"` in the console on initial load. It works as intended once I start checking items; it's just the initial load that's problematic. I'll see what else I can find out.
